packages/eslint-plugin-pf-codemods/src/rules/v6/avatarReplaceBorderForIsBordered/avatar-replace-border-for-isBordered.ts

```diff
@@ -0,0 +1,16 @@
+import { getFromPackage, renameProps } from '../../helpers';
+
+// https://github.com/patternfly/patternfly-react/pull/9954
```

Comment:
Similar to the README comment, can we just add a link to the core bump PR in a comment as well